cbsurge/components/electricity/__init__.py

```diff
@@ -137,14 +139,21 @@ def __call__(self, *args, **kwargs):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
+        project = Project(path=os.getcwd())
+        self.local_path = project.geopackage_file_path
+
+    @property
+    def affected_layer(self):
+        return f"{self.component}.affected"
 
     @property
-    def affected_name(self):
-        return f'{self.name}_affected'
+    def affected_variable(self):
+        return f"{self.name}.affected"
 
     @property
-    def percentage(self):
-        return self.__dict__.get('percentage', False)
+    def affected_percentage_variable(self):
+        return f"{self.name}.affected_percentage"
```

Comment:
Use `_affected_percentage`

If you think connecting words by `.` is better, please update other components to use same name. All components should use the same naming pattern.